documentation-website/Writerside/topics/Table-Definition.md

```diff
@@ -146,42 +194,63 @@ val name = varchar("name", 50).uniqueIndex()
+For example, the `IntIdTable` by default has an auto-incrementing integer column, `id`, which is defined as the primary key.
+
+An `IdTable` that requires a primary key with multiple columns can be defined using `CompositeIdTable`.
+In this case, each column that is a component of the table's `id` should be identified by `entityId()`:
+```kotlin
+object Towns : CompositeIdTable("towns") {
+    val areaCode = integer("area_code").autoIncrement().entityId()
+    val latitude = decimal("latitude", 9, 6).entityId()
+    val longitude = decimal("longitude", 9, 6).entityId()
+    val name = varchar("name", 32)
+
+    override val primaryKey = PrimaryKey(areaCode, latitude, longitude)
+}
+```
+
 ### Foreign Key
 
-The `FOREIGN KEY` SQL constraint links two tables. Foreign key is a column from one table that refers to the primary key 
-or columns with a unique index from another table. To configure the foreign key, use `reference()` or `optReference()` 
-method. The second one let the foreign key accept the `null` value.
+The `FOREIGN KEY` SQL constraint links two tables. A foreign key is a column from one table that refers to the primary key 
+or columns with a unique index from another table. To configure a foreign key on a column, use `reference()` or `optReference()` 
+methods. The latter lets the foreign key accept a `null` value. To configure a foreign key on multiple columns,
+use `foreignKey()` directly within an `init` block.
 
 `reference()` and `optReference()` methods have several parameters:
 
-* `name: String` is a name for foreign key column, which will be used in actual SQL queries.
+* `name: String` is a name for the foreign key column, which will be used in actual SQL queries.
 * `ref: Column<T>` is a target column from another parent table.
-* `onDelete: ReferenceOption? = null` is an action to the case when linked row from a parent table can be deleted.
-* `onUpdate: ReferenceOption? = null` is an action to the case when value in a referenced column can be changed.
-* `fkName: String? = null` is a foreign key constraint name.
+* `onDelete: ReferenceOption? = null` is an action for when a linked row from a parent table will be deleted.
+* `onUpdate: ReferenceOption? = null` is an action for when a value in a referenced column will be changed.
+* `fkName: String? = null` is a name for the foreign key constraint.
 
-Enum class `ReferenceOption` has four values:
+Enum class `ReferenceOption` has five values:
 
-* `RESTRICT` is an option that restricts changes on a referenced column, and the default option for MySQL dialect.
-* `NO_ACTION` is the same as RESTRICT, and the default option for Oracle and SQL Server dialects.
+* `RESTRICT` is an option that restricts changes on a referenced column, and the default option for most dialects.
+* `NO_ACTION` is the same as RESTRICT in some, but not all, databases, and the default option for Oracle and SQL Server dialects.
```

Comment:
Here and in any other places where we have lists of options/properties, I think it would be nice to start using the [`<deflist>`](https://www.jetbrains.com/help/writerside/lists.html#type-wide) element from Writerside. It can be in a separate PR though.

Reply:
Sure, I will at least use it in the places I have changes already.

Reply:
That's ok. It also makes sense as the file is already in Markdown. I think in XML you'd have to use the `<code>` element in the title.